connect-examples/server-library-example/README.md

```diff
@@ -0,0 +1,133 @@
+# Spark Server Library Example - Custom Datasource Handler
```

Comment:
`Spark Server` -> `Spark Connect Server`?

Reply:
This follows the naming from https://github.com/apache/spark/pull/48922